queries.sql

```diff
@@ -42,3 +42,33 @@ SELECT species, MAX(weight_kg) AS maximum_weight, MIN(weight_kg) AS minimum_weig
 
 SELECT species, ROUND(AVG(escape_attempts), 2) AS average_escape_attempts FROM animals WHERE date_of_birth >= '1990-01-01' AND date_of_birth < '2000-12-31' GROUP BY species;
 
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = 1 WHERE name LIKE 'Agumon';
+
+UPDATE animals SET owner_id = 2 WHERE name LIKE 'Gabumon' OR name LIKE 'Pikachu';
+
+UPDATE animals SET owner_id = 3 WHERE name LIKE 'Devimon' OR name LIKE 'Plantmon';
+
+UPDATE animals SET owner_id = 4 WHERE name LIKE 'Charmander' OR name LIKE 'Squirtle' OR name LIKE 'Blossom';
+
+UPDATE animals SET owner_id = 5 WHERE name LIKE 'Angemon' OR name LIKE 'Boarmon';
```

Comment:
kindly move the update statements inside the data.sql, since everything that populates tables with data should be inside data.sql as mentioned in the requirements:   ✔😀✔ 👍

![image](https://user-images.githubusercontent.com/87448628/178096385-c757a687-f11c-45f4-aaad-8cec4ffe5739.png)